ocular/tests/single_node_chain_txs.rs

```diff
@@ -249,93 +246,6 @@ fn local_single_node_chain_test() {
     .expect("Could not parse sign doc.");
     let _expected_msg_exec_raw = expected_msg_exec_sign_doc.sign(&sender_private_key);
 
-    // Automated tx handler delegated workflow
```

Comment:
Note: Test node is unable to process gRPC requests and so account number and sequence number fetches are not possible making these tests irrelevant so they've been removed.